examples/vue/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+lerna-debug.log*
+
+node_modules
+dist
+dist-ssr
+*.local
+
+# Editor directories and files
+.vscode/*
+!.vscode/extensions.json
+.idea
+.DS_Store
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
+# Ignore .env files containing sensitive data
+.env.local
+.env.*.local
```

Comment:
.env.local
.env.*.local

These are not needed as you already have *.local

Reply:
.gitignore update is done